Update new Desc file and Data Model

huyanh0720 opened the pull request:
- Add new table Dim_Date with attributes: Date_ID, Appointment_Date, Day, Month, Year, Quarter, Day_Of_Week.
- Delete Appointment_Date from Diagnosis table.
- Add Date_ID and Appointment_Date to Medical table.

huyanh0720 committed:
Update new Desc file and Data Model

- Add new table Dim_Date with attributes: Date_ID, Appointment_Date, Day, Month, Year, Quarter, Day_Of_Week.
- Delete Appointment_Date from Diagnosis table.
- Add Date_ID and Appointment_Date to Medical table.

duy-deliver reviewed (commented):
```
- Recreate model from Warehouse to Database.
- Create Medical_Detail table.
- Delete Diagnosis_ID from Medical table.
- Fix Medical table and Date_Time table.
- Delete Diagnosis table.
- Keep tables: Disease, Symptom, Insurance, Patient.
```

huyanh0720 committed:
Updated Desc and Model.

- Recreate model from Warehouse to Database.
- Create Medical_Detail table.
- Delete Diagnosis_ID from Medical table.
- Fix Medical table and Date_Time table.
- Delete Diagnosis table.
- Keep tables: Disease, Symptom, Insurance, Patient.

duy-deliver reviewed (approved):
Ok
